<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>volume3DOptions</class>
 <widget class="QWidget" name="volume3DOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Interpolation Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="interpolateComboBox">
       <item>
        <property name="text">
         <string>Nearest Neighbour</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Linear</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cubic</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>3-Slice</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkAxial3D">
          <property name="text">
           <string>Axial 3D</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkSagittal3D">
          <property name="text">
           <string>Sagittal 3D</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkCoronal3D">
          <property name="text">
           <string>Coronal 3D</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Axis Flips</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="flipXCheck">
          <property name="text">
           <string>Flip X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="flipYCheck">
          <property name="text">
           <string>Flip Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="flipZCheck">
          <property name="text">
           <string>Flip Z</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupRayCastVolume">
     <property name="title">
      <string> Ray-Cast Volume</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioComposite">
        <property name="text">
         <string>Composite</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioMIP">
        <property name="text">
         <string>MIP</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioIsosurface">
        <property name="text">
         <string>Isosurface</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Piecewise Function:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="comboPieceFunc"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Colour Transfer Function: </string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="comboCTFunc"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
